apps/io_tester/io_tester.cc

```diff
@@ -968,7 +968,7 @@ int main(int ac, char** av) {
             if (*st_type == directory_entry_type::directory) {
                 auto fs = file_system_at(storage).get0();
                 if (fs != fs_type::xfs) {
-                    throw std::runtime_error(format("This is a performance test. {} is not on XFS", storage));
+                    std::cout << "WARNING!!! This is a performance test. " << storage << " is not on XFS" << std::endl;
```

Comment:
does it make sense to send this message to `std::cerr` ? and to use
```c++
fmt::print(cerr, "This is a performance test. {} is not on XFS\n", storage);
```
less quotes this way.

Reply:
bonus, could even use something like
```c++
fmt::print(cerr, fmt::emphasis::bold | fg(fmt::color::red), "WARNING!!!");
fmt::print(cerr, "This is a performance test. {} is not on XFS\n", storage);
```

Reply:
The whole io-tester uses old-style cout<< printing. I think it deserves its own PR to convert everything into fmt

Reply:
I've my /home ext4-formatted and when I want to quickly validate some io-related change, I run a short and stupid io-tester work. And I'm sick and tired of need to work this check around :(

Reply:
i see. btrfs formatted here. so i am already in your shoes =)